Classes/Command/EventStoreCommandController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Neos\Cqrs\Command;
```

Comment:
yes, separate issue. Maybe we'll need to put the actual logic into a sub namespace and the infrastructural code (CLI etc) into `Application`.